src/gui/abstractfileitemwidget.cpp

```diff
@@ -38,10 +38,13 @@ static const int boxVSpacing = 10;
 static const int shadowBlurRadius = 20;
 static const QSize iconSize = QSize(15, 15);
 
+static const QColor folderIconColorLight = QColor("#666666");
```

Comment:
```suggestion
static const QColor folderIconColorLight = QColor(0x666666);
```

`Qt` Creator recommends this int-based constructor because it is faster. It doesn't matter in this case, but it would demonstrate the principle and serves as an example for other constructor calls.